pages/resources/quizzes/FirstLang/FirstProgrammingLanguageQuiz.tsx

```diff
@@ -65,9 +59,21 @@ const FirstProgrammingLanguageQuiz = () => {
   };
 
   const handleOptionClick = (option: QuizOptionViewState) => {
-    // TODO create a new quizViewState and then call setQuizViewState
-    console.log(option, quizViewState);
-    // TODO generate a new quiz state based off the old state plus the clicked option
+    const selectedQuizOption = quizViewState.questions.map((question) => ({
```

Comment:
tiny nitpick, it could be helpful to comment above this code about what the function is doing, what are its inputs outputs etc. just thinking of how to consolidate your knowledge and help out a new dev who may be working on your components.

Reply:
I was under the impression that our commits should not contain commented code.

Reply:
you shouldn't comment out code and then merge that into main. But I think Lucky is just suggesting you use normal comments to better explain what you're doing. Using comments to explain your code is definitely allowed and encouraged!

Reply:
oh sweet! good to know guys!